EDI/Operation/02_ManageHeaderFileSett.md

```diff
@@ -0,0 +1,404 @@
+   The *Exporter definition sources* view is displayed.
+
+9. Continue with [Specify message dates](#specify-message-dates) described below.
+
+
+
+## Specify message dates
+
+Specify the message dates your business partner requires.   
+
+Each invoice can have a lot of different dates, for example the invoice date, the expected delivery date, the tax point date, and much more. The Actindo EDIFACT message structure contains only one BTM segment in its raw state. However, for many business partners, multiple BTM segments must be completed. As soon as you have specified one value in a BTM segment, a further BTM segment is added automatically so that you can specify as much BTM segments as your business partner require.
+
+This procedure shows how to edit message dates. The message dates require both the description of the date and a variable used to determine it.
+
+#### Prerequisites
+
+- You have expanded the message structure, see [Get header file segments](#get-header-file-segments).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Select message definition > File settings tab > Expand message structure*
+
+![Message structure](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDates.png "[Message structure]")
+
+1. Expand the *S_DTM* (DATE &frasl; TIME &frasl; PERIOD) segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it.   
+    The *S_DTM* segment in turn consists of the subordinate *C_C507* segment.
+
+2. Expand the *C_C507* (Date and &frasl; or time, or period relevant to the) segment by clicking the ![Collapsed](../../Assets/Icons/NodeCollapsed.png "[Collapsed]") (Collapsed) button at it. In most cases it is required to specify this segment.   
+    The *C_C507* segment has been expanded. The individual fields of this segment are displayed.
+
+    ![C_C507 segment](../../Assets/Screenshots/EDI/Operation/ExportDefMessDefDTMSegment.png "[C_C507 segment]")
+
+3. Specify the *D_2005 (Code qualifying the function of a date, time or)* field. In most cases, a constant value is required here. To do this, click the ![Set constant value](../../Assets/Icons/Edit04.png "[Set constant value]") (Set constant value) button to the right of the *D_2005* row.  
+    The *Edit text value* pop-up window is displayed.
+
+    ![Edit text value](../../Assets/Screenshots/EDI/Operation/ExportDefEditTextValue.png "[Edit text value]")
+ 
+4. Enter the required text value and click the [SAVE] button.  
+    A new *S_DTM* segment has been added below the *S_DTM* segment you are currently working on.
+
+5. Specify the *D_2380 (The value of a date, a date and time, a time or of a period)* field. In most cases, a string attribute is required here. To do this, click the ![Add string attribute](../../Assets/Icons/Plus03.png "[Add string attribute]") (Add string attribute) button to the right of the *D_2380* row.    
```

Comment:
Muss man auch D2380 und D2379 einstellen oder kann man? In der Definition steht nicht M (Mandatory). Könnte man evtl. diese zwei unter C_C507 unterbringen oder erklären, dass man die einstellen muss "if desired"?

Reply:
Ich erkläre ja nur generelle Beispiele und es gibt keinen Anspruch auf Vollständigkeit